src/ReverseInteger.java

```diff
@@ -0,0 +1,26 @@
+public class ReverseInteger {
+    public static int reverse(int x) {
+        long res = 0;
+        int mod = Math.abs(x);
+        int i = (int) Math.log10(mod);
+        if(i == 0 && (mod > 10 || mod < -10)) {
+            return 0;
+        }
+        while(x != 0 && i >= 0) {
+            res += (x % 10) * Math.pow(10, i);
+            x /= 10;
+            i--;
+        }
+        if(res > 2147483647 ||
```

Comment:
Use Integer.MAX_VALUE and Integer.MIN_VALUE constants

Reply:
Okay